improver_tests/calibration/dz_rescaling/test_estimate_dz_rescaling.py

```diff
@@ -30,19 +30,22 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Unit tests for the EstimateDzRescaling plugin."""
 
+from datetime import datetime as dt
+from datetime import timedelta
 from typing import List
 
 import iris
 import numpy as np
-import pandas as pd
 import pytest
-from iris.coords import AuxCoord, DimCoord
-from iris.cube import Cube
+from iris.cube import Cube, CubeList
 
 from improver.calibration.dz_rescaling import EstimateDzRescaling
-from improver.metadata.constants.time_types import TIME_COORDS
-from improver.spotdata.build_spotdata_cube import build_spotdata_cube
+from improver.metadata.constants.time_types import DT_FORMAT
 from improver.spotdata.neighbour_finding import NeighbourSelection
+from improver.synthetic_data.set_up_test_cubes import (
+    set_up_spot_percentile_cube,
+    set_up_spot_variable_cube,
+)
```

Comment:
I think that the `forecast_data` and `truth_data` global variables can now be removed, as they're already being defined in the functions that use them. I think that potentially the definition of the other global variables could be moved to being within `_create_neighbour_cube`.

Reply:
Done. Retained `wmo_id` as a global, but capatalised.